apps/api/src/migration/1663698520477-RemoveDuplicateMilestones.ts

```diff
@@ -34,17 +34,22 @@ export class RemoveDuplicateMilestones1663698520477 implements MigrationInterfac
       .from('ien_applicant_status')
       .where({ id: In(status_id_list) })
       .execute();
-
-    await queryRunner.manager
-      .createQueryBuilder()
-      .insert()
-      .into('ien_applicant_status')
-      .values({
-        id: '3fd4f2b0-5151-d7c8-6bbc-3a0601b5e1b0',
-        status: 'Candidate Withdrew from Competition',
-        category: 'IEN Recruitment Process',
-      })
-      .execute();
+    try {
+      await queryRunner.manager
+        .createQueryBuilder()
+        .insert()
+        .into('ien_applicant_status')
+        .values({
+          id: '3fd4f2b0-5151-d7c8-6bbc-3a0601b5e1b0',
```

Comment:
Why do we need to insert this? I see it on my local database.

Reply:
I had to add this because this migration was breaking in the pipeline. I think the query builder referenced the entity when constructing the query. By adding the version column to the entity, this migration tried to add a version to the insert query. I patched it by using a direct query instead. 
![Screenshot 2023-08-14 at 3 51 45 PM](https://github.com/bcgov/internationally-educated-nurses/assets/54554766/3850f9b8-83c0-427f-b21f-94aa68a63db2)